node/network/protocol/src/request_response/outgoing.rs

```diff
@@ -32,6 +32,9 @@ pub enum Requests {
 	ChunkFetchingV1(OutgoingRequest<v1::ChunkFetchingRequest>),
 	/// Fetch a collation from a collator which previously announced it.
 	CollationFetchingV1(OutgoingRequest<v1::CollationFetchingRequest>),
+	/// Fetch a collation from a collator which previously announced it.
+	/// Compared to V1 it requires specifying which candidate is requested by its hash.
+	CollationFetchingVStaging(OutgoingRequest<vstaging::CollationFetchingRequest>),
```

Comment:
style: I'd place this at the bottom to create a clean separation between V1 and VStaging requests